GVFS/GVFS.Platform.Mac/MacProductUpgraderPlatformStrategy.cs

```diff
@@ -21,8 +21,7 @@ public override bool TryPrepareLogDirectory(out string error)
 
         public override bool TryPrepareApplicationDirectory(out string error)
         {
-            string rootDirectoryPath = ProductUpgraderInfo.GetUpgradesDirectoryPath();
-            string toolsDirectoryPath = Path.Combine(rootDirectoryPath, ProductUpgrader.ToolsDirectory);
+            string toolsDirectoryPath = ProductUpgraderInfo.GetUpgradeApplicationDirectory();

```

Comment:
Why do we call it `toolsDirectoryPath` but we are getting the `ApplicationDirectory`?  Do `tools == Application` or are the tools just located in the Application directory?

Reply:
`Tools` was how it was referred to previously. I think `UpgradeApplication` is a bit more descriptive now - I will go through and update the terminology to be more consistent.